### PR TITLE
Backport types internal (SMX-120,SMX-143)

### PR DESCRIPTION
**Pull request by hypervillain**
- Use alpha (latest) version of @prismicio/types-internal, @prismicio/mocks editor-fields and editor-ui everywhere
- Replace types with types-internal everywhere

### CODE REVIEW

**Comment by arnaudlewis** [comment]
Don't forget the backport the rest regarding the mock config of date and link that are new in editor 0.3.0.

File: package.json [hunk 1]
```diff
@@ -42,6 +42,7 @@
     "prettier": "^2.8.4"
   },
   "dependencies": {
+    "@prismicio/mocks": "^2.0.0-alpha.2",
```
Comment (lihbr) [inline package.json]: Why does `@prismicio/mocks` needs to be installed top level?

File: packages/manager/test/lib-mockSlice.test.ts [hunk 2]
```diff
@@ -31,13 +28,13 @@ describe("mockSlice", () => {
 		};
 		const geoR = GeoPointContent.decode(geo.key);
 
-		expect(isRight(geoR)).toBeTruthy();
+		expect(isRight(geoR)).toBeFalsy();
```
Comment (lihbr) [inline packages/manager/test/lib-mockSlice.test.ts]: This should be truthy no? You likely need to uncomment line 25

Left as is (as I understand it) it defeats the purpose of the test (which also has no reason to be a test in first place here, we shouldn't have to test types-internal here)

File: packages/slice-machine/components/Select/index.tsx [hunk 3]
```diff
@@ -13,18 +13,18 @@
...
+//   }
 
-  // @ts-expect-error - `default` is added by Rollup during tests since
-  // `react-select` has default and named exports. It does not exist normally.
-  Select = _Select.default;
-}
+//   // @ts-expect-error - `default` is added by Rollup during tests since
+//   // `react-select` has default and named exports. It does not exist normally.
+//   Select = _Select.default;
+// }
```
Comment (lihbr) [inline packages/slice-machine/components/Select/index.tsx]: If this is not needed anymore, maybe you can just get rid of it?

**Review by lihbr — CHANGES_REQUESTED** [review]
LGTM, just some small things.

Could you have a look at backporting the date & link from editor 0.3.0 as Arnaud mentioned? (this PR or another, whatever works best for you)

**Review by lihbr — APPROVED** [review]

**Pull request merged**